tools/deployment/make_linux_package.sh

```diff
@@ -51,7 +52,7 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
+  
```

Comment:
I should have my return key taken away from me until I learn. Or whichever finger keeps hitting it surgically shortened...